packages/node_modules/@webex/plugin-meetings/src/members/collection.js

```diff
@@ -68,15 +68,16 @@ export default class MembersCollection {
   }
 
   /**
-   * @param {Array} participants
+   * @param {Object} locus
    * @returns {Object} refreshed complete list
    */
-  refresh(participants) {
-    if (!participants) {
+  refresh(locus) {
+    if (!locus) {
```

Comment:
After looking into `constructMembers` function, I don't think this check is necessary.

Reply:
Sure. I can check for locus.participants